test/parallel/test-crypto-sign-verify.js

```diff
@@ -71,6 +71,38 @@ const keyPem = fs.readFileSync(common.fixturesDir + '/test_key.pem', 'ascii');
   assert.strictEqual(verified, true, 'sign and verify (stream)');
 }
 
+{
+  [ 'RSA-SHA1', 'RSA-SHA256' ].forEach((algo) => {
+    [ null, -2, -1, 0, 16, 32, 64 ].forEach((saltLength) => {
+      let verified;
+
+      // Test sign and verify with the given parameters
+      const s4 = crypto.createSign(algo)
+                       .update('Test123')
+                       .sign(keyPem, null, { padding: 'pss', saltLength });
+      verified = crypto.createVerify(algo)
+                       .update('Test')
+                       .update('123')
+                       .verify(certPem, s4, { padding: 'pss', saltLength });
+      assert.strictEqual(verified, true, 'sign and verify (buffer, PSS)');
+
+      // Setting the salt length to -2 should always work for verification
+      verified = crypto.createVerify(algo)
+                       .update('Test123')
+                       .verify(certPem, s4, { padding: 'pss', saltLength: -2 });
+      assert.strictEqual(verified, true, 'sign and verify (buffer, PSS)');
+    });
+  });
+}
+
```

Comment:
There are test vectors for RSA-PSS in the link of https://www.emc.com/emc-plus/rsa-labs/standards-initiatives/pkcs-rsa-cryptography-standard.htm . We can make verify tests by using this test vectors. 
If you cannot get the test vector file, it is in https://github.com/shigeki/ohtsu_rsa_pss_js/blob/master/tests/vectors/pss-vect.txt. Please refer related test vectors of RSA-PSS(example1 for RSA1024bits, and example10 for RSA2048bits) in JavaScript are in https://github.com/shigeki/ohtsu_rsa_pss_js/tree/master/tests/ .

Unfortunately, openssl cannot accept explicit salt. So we can make sign tests for RSA-PSS by writing signature data to files and verifying its signature by using `openssl dgst` command with `-signopt rsa_padding_mode:pss`.